docs/v2-system-addon/snippets.md

```diff
@@ -30,7 +30,7 @@ async function setFoo() {
 
 `.blockList`: (Array) An array of snippet IDs that have been blocked by the users, or an empty array if none have been blocked.
 
-`.blockSnippetById(id)`: (func) A function that adds an id to a blockList and returns
+`.blockSnippet(id)`: (func) A function that adds an id to a blockList and returns
```

Comment:
yep this sounds fine to me, we can bump the version if a more major change is made

Reply:
As stated in https://github.com/mozmeao/snippets-service/pull/365 from the snippets-service side it's better that we keep the same name.